ui/packages/app/grafana-panel-plugin/src/plugin.json

```
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
+  "type": "panel",
+  "name": "Parca Panel",
```

```suggestion
  "name": "Parca Flamegraph",
```
I'd call it "Parca" or simply "Parca Flamegraph".